lib/jekyll-gfm-admonitions.rb

```diff
@@ -90,10 +96,10 @@ def process_doc(doc)
     end
 
     def convert_admonitions(doc)
-      doc.content.gsub!(/>\s*\[!(IMPORTANT|NOTE|WARNING|TIP|CAUTION)\]\s*(.*)\s*\n((?:>.*\n?)*)/) do
+      doc.content.gsub!(/>\s*\[!(IMPORTANT|NOTE|WARNING|TIP|CAUTION)\]([^\n]*)\n((?:>.*\n?)*)/) do
```

Comment:
Why don't you add '^' like `doc.content.gsub!(/^>\s*\[!(IMPORTANT|NOTE|WARNING|TIP|CAUTION)\]([^\n]*)\n((?:>.*\n?)*)/) do`?
https://manabu-nakamura.github.io/jekyll-gfm-admonitions/
https://github.com/Helveg/jekyll-gfm-admonitions/compare/main...manabu-nakamura:jekyll-gfm-admonitions:main

Reply:
Ah, I see what you mean. I think that would break cases where the block quote doesn't start at the beginning of the line.

> BlockQuote 1

    > BlockQuote 2 is indented


But maybe I can adjust it.

`^\s*?>`